libnopegl/src/text.h

```diff
@@ -56,6 +66,9 @@ struct text {
     int32_t height;
     struct darray chars; // struct char_info
     struct texture *texture;
```

Comment:
Not related but would it make sense to rename `texture` to `atlas`, `texture` seems too generic here.

Reply:
Yeah I might but I'm a bit concerned with the potential confusion with the `struct atlas` and its API. I added a commit to rename it to `atlas_texture` instead.

Reply:
Perfect !